### ISSUE TITLE
Spambot on Non offline servers

### ISSUE THREAD

**Issue by GlitchHead** [DarkStorm652/DarkBot#37]
Well first off, when using the Spambot on non offline servers with account list and proxy list a lot of @$!% happens a REALLY lot of spam and a lot of "account failed" I'm not talking about proxy's not working or alts not working i have 40 accounts logged in but when i press join only 2 join the server!?! with the delay at 5000 and most of the joins give me "account failed" Not that the username or password is wrong.
The main reason im posting this is this spam bot called "Apocalypse Bot" does not even use proxy's "they don't work" and it has premium accounts join very fast on servers without connection throttled "you can just change the throttle delay to 6000 on Apocalypse Bot" now I'm not here to say darkbot sucks and this other bot thing is better its just that this other spam bot Apocalypse Bot can join servers without http proxy's or socks at a fast speed and darkbot requires proxy's.... I'm just a bit confused on how apocalypse Bot is working with out getting you blacklisted from mojang login for 30 mins.....
Im going to include the program "ApocalypseBot.jar" if you would like to scan the program and see what it does and maby use its methods in darkbot the only reason me wanting this to be in darkbot is so i can have 50 bots following me around and bypassing the "you cannot chat until you move!" thing..
I hope you found this interesting if this could be added or copied into darkbot that would be wonderful! 

download link to Apocalypse bot (made by Klintos) GUI only
https://dl.dropboxusercontent.com/s/ba05dyv9rgr1fvk/ApocalypseBot.jar

**Comment by Vincentyification** [comment]
Not an issue, frankly a terribly put feature request of a harmful feature (which does no good to anyone ever). Requesting close, @DarkStorm652.

**Comment by DarkStorm652** [comment]
Let me remind you that support for a chat flooding mechanism is only preliminary and not generally supported at the moment. Everything in the `org.darkstorm.darkbot.mcwrapper` package is intended more as example uses of the API than anything else. With that said, I will be revisiting the implementation of everything there more substantially as a separate (dependent) project some time in the future- but expect that to come later after major fixes to this project.

Also, anything and everything you see output to standard output is _all_ meaningless debug information as a general rule.

On a side note, you mention the need for proxies -- the reason for the need for login proxies is that if you have an enormous list of accounts of which a good 40% fail to log in, if you do not provide a decent number of proxies, you will wind up blacklisting your own IP address from Mojang authentication servers for a period of 10 minutes. This requirement is made mandatory to avoid confusion, although the blacklisting for failed logins generally won't affect those with lists consisting of mostly (90+%) valid accounts. "Account failed" is a debug message indicating that any of (a) invalid credentials, (b) a failure to establish a connection to an MC server, or (c) a disconnection from an MC server have occurred. SOCKS proxies for Minecraft connections are not, on the other hand, required at all, and reliable proxies of this type are decently difficult to find. ApocalypseBot utilizes MCProtocolLib, which has a much more stable networking system that uses Netty and does not rely on a ticking thread. The decision to design DarkBot around the same time-keeping system as Minecraft has proven itself repeatedly to be a terrible one and a constant thorn in my side.

tl;dr: don't expect to use a heavyweight project for a very simple task effectively.

**Comment by GlitchHead** [comment]
DarkStorm652 Whats your method on getting good proxies?

**Comment by TheBiblMan** [comment]
why are you posting that here?

**Comment by Unihedro** [comment]
@GlitchHead http://stackoverflow.com/questions/ask?title=plz%20halp%20me%20urgent%20spambot%20proxy%20problem Be sure to leave me a link so I can close - I mean review it.

**Comment by GlitchHead** [comment]
No thanks.
